packages/pn-personafisica-webapp/src/navigation/__test__/navigation.utility.test.ts

```diff
@@ -1,11 +1,13 @@
 import { vi } from 'vitest';
+
 import { EventPageType } from '@pagopa-pn/pn-commons';
 
 import { getConfiguration } from '../../services/configuration.service';
 import { getCurrentEventTypePage, goToLoginPortal } from '../navigation.utility';
 import { APP_STATUS, DELEGHE, DETTAGLIO_NOTIFICA, NOTIFICHE, RECAPITI } from '../routes.const';
 
 const replaceFn = vi.fn();
```

Comment:
If it is not used anymore, remove it. Do this for all tests below

Reply:
What are you reffering to?

Reply:
to replaceFn